### PR TITLE
Feature/4271 add population data to communities

### LINKED ISSUE #4271: Add population data to Communities
### Situation Description

Right now, population data can only be imported for regions and districts. However, especially for the statistics page, it can sometimes be useful to also have population data on a community-basis.

### Feature Description

- [ ] Add column `community_id` to `populationdata`
- [ ] Add column `Community` in import&export template, which can be empty optionally
- [ ] If a region&district filter is set in statistics view, show all communities with their respective incidence

### Additional Information

If working on this, one may also consider implementing #3984 by either adding a small table showing the most important population data information (Region, District, Community, Growthrate and total Population) or improving the notice to be more clear.

This Issue is based on a user request.

### PR DESCRIPTION
**Pull request by arrol**
Added Possibilty to import/export populationData on community base.
Incidence can now be calculated for communitys, in Statistic View.

----
Edit: closes #4271

### CODE REVIEW

File: sormas-backend/src/main/resources/sql/sormas_schema.sql [hunk 1]
```diff
@@ -7078,4 +7078,9 @@ $BODY$;
 
 INSERT INTO schema_version (version_number, comment) VALUES (353, 'Provide SQL function to generate a UUIDv4 encoded as base32 #4805');
 
+-- 2021-03-07 Add Community Reference to PopulationData entity #4271
+ALTER TABLE populationdata ADD COLUMN community_id bigint;
+ALTER TABLE populationdata ADD CONSTRAINT fk_populationdata_community_id FOREIGN KEY (community_id) REFERENCES community(id);
+ALTER TABLE community ADD COLUMN growthRate real;
+INSERT INTO schema_version (version_number, comment) VALUES (354, 'Add Community Referenz to PopulationData entity #4271');
```
Comment (MateStrysewske) [inline sormas-backend/src/main/resources/sql/sormas_schema.sql:7085]: Referenz => reference

File: sormas-ui/src/main/java/de/symeda/sormas/ui/importer/PopulationDataImporter.java [hunk 2]
```diff
@@ -88,14 +91,32 @@ protected ImportLineResult importDataFromCsvLine(
 					district = districts.get(0);
 				}
 			}
+			if (PopulationDataDto.COMMUNITY.equalsIgnoreCase(entityProperties[i])) {
+				if (DataHelper.isNullOrEmpty(values[i])) {
+					community = null;
+				} else {
+					List<CommunityReferenceDto> communitys = FacadeProvider.getCommunityFacade().getByName(values[i], district, false);
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/importer/PopulationDataImporter.java:98]: communitys => communities

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Hey! Sorry that the review took so long. I don't see any problems with the code so far. Can you please correct the two naming issues and resolve the merge conflict? I'll then test the feature and merge it if everything works fine.

**Comment by arrol** [comment]
No problem for me, David told me already, that the timing for the last sprint, would be pretty close :)
Typo's and merge conflict should be fixed now.

**Review by MateStrysewske — APPROVED** [review]

**Comment by MateStrysewske** [comment]
Thanks! Works fine!

**Pull request merged**